datasync/chaindatafetcher/kas/repository_traces.go

```diff
@@ -0,0 +1,197 @@
+package kas
+
+import (
+	"fmt"
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/blockchain/vm"
+	"github.com/klaytn/klaytn/common"
+	"reflect"
+	"strings"
+)
+
+func isInternalTxResult(trace *vm.InternalTxTrace) bool {
+	emptyResult := &vm.InternalTxTrace{
```

Comment:
It would be better if `emptyResult` is declared as a const variable.

Reply:
Applied. 5dc244f38ea6698385f1815ed5f9658177e80848